<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кинотеатр</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_plan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>611</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>План зала</string>
    </property>
    <widget class="QTableWidget" name="tw_hall_plan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>591</width>
       <height>440</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_cinemas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>210</height>
     </rect>
    </property>
    <property name="title">
     <string>Кинотеатры</string>
    </property>
    <widget class="QListWidget" name="lw_cinemas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_halls">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>300</width>
      <height>210</height>
     </rect>
    </property>
    <property name="title">
     <string>Кинозалы</string>
    </property>
    <widget class="QListWidget" name="lw_halls">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_sessions">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>300</width>
      <height>210</height>
     </rect>
    </property>
    <property name="title">
     <string>Сеансы</string>
    </property>
    <widget class="QListWidget" name="lw_sessions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_info">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>300</width>
      <height>470</height>
     </rect>
    </property>
    <property name="title">
     <string>Информация о сеансе</string>
    </property>
    <widget class="QTextBrowser" name="tb_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>440</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Главная</string>
    </property>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Кинотеатр</string>
    </property>
    <addaction name="act_add_cinema"/>
    <addaction name="act_del_cinema"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Кинозал</string>
    </property>
    <addaction name="act_add_hall"/>
    <addaction name="act_del_hall"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Сеанс</string>
    </property>
    <addaction name="act_add_session"/>
    <addaction name="act_del_session"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Фильм</string>
    </property>
    <addaction name="act_add_film"/>
    <addaction name="act_del_film"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>Поиск</string>
    </property>
    <addaction name="act_find_session"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="act_add_cinema">
   <property name="text">
    <string>Добавить кинотеатр</string>
   </property>
  </action>
  <action name="act_add_hall">
   <property name="text">
    <string>Добавить кинозал</string>
   </property>
  </action>
  <action name="act_add_session">
   <property name="text">
    <string> Добавить сеанс</string>
   </property>
  </action>
  <action name="act_add_film">
   <property name="text">
    <string>Добавить фильм</string>
   </property>
  </action>
  <action name="act_del_cinema">
   <property name="text">
    <string>Удалить кинотеатр</string>
   </property>
  </action>
  <action name="act_del_hall">
   <property name="text">
    <string>Удалить кинозал</string>
   </property>
  </action>
  <action name="act_del_session">
   <property name="text">
    <string>Удалить сеанс</string>
   </property>
  </action>
  <action name="act_del_film">
   <property name="text">
    <string>Удалить фильм</string>
   </property>
  </action>
  <action name="act_find_session">
   <property name="text">
    <string>Ближайший сеанс</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
